projects/1.20-fabric/assets/dynamic-surroundings/dsurround/lang/zh_cn.json

```
@@ -0,0 +1,196 @@
+{
+  "dsurround.modname": "动态环绕",
+  "dsurround.newversion.update": "有更新可用",
+  "dsurround.newversion.downloadpage": "点击进入下载界面",
+  "dsurround.newversion.curseforge": "CurseForge",
+  "dsurround.newversion.modrinth": "Modrinth",
+  "dsurround.newversion.releasenotes": "更新日志",
+  "dsurround.newversion.releasenotespage": "点击查看更新日志",
+  "dsurround.mod.link.sources": "源代码存储库",
+  "dsurround.mod.link.credits": "声音资源的版权和归属",
+  "dsurround.mod.link.clothapi": "嵌入: Cloth API (Fabric)",

```

```suggestion
  "dsurround.mod.link.clothapi": "嵌入：Cloth API (Fabric)",
```
汉语标点请使用规范
英文内的标点直接保留

---

改啦